pkg/apis/configuration/validation/validation_test.go

```diff
@@ -237,6 +306,48 @@ func TestValidateUpstreamsFails(t *testing.T) {
 	}
 }
 
+func TestValidateNextUpstream(t *testing.T) {
+	tests := []struct {
+		inputS string
+	}{
+		{
+			inputS: "error timeout",
+		},
+		{
+
+			inputS: "http_404 timeout",
+		},
+	}
+	for _, test := range tests {
+		allErrs := validateNextUpstream(test.inputS, field.NewPath("next-upstreams"))
+		if len(allErrs) > 0 {
+			t.Errorf("validateNextUpstream() returned errors %v for valid input for the case of %s", allErrs, test.inputS)
```

Comment:
consider printing `validateNextUpstream("error timeout")` similar to the other tests. For example, see https://github.com/nginxinc/kubernetes-ingress/blob/e6d5cf47761f31d1218c688314b02880ed3c7e3b/pkg/apis/configuration/validation/validation_test.go#L67

Reply:
will do, note there are a good few tests in the code that don't follow this practice

Reply:
will do, note many other tests don't follow this practice

Reply:
yep. the other tests have more complicated arguments (structs) that are not suitable for printing  like that